scrapper.py

```diff
@@ -58,6 +66,16 @@ def validate_config(crawler_path):
     pass
 
 
+def main():
+    html = 'https://vestnik.lunn.ru/arhiv-zhurnala/2021-god/vypusk-56-iv-kvartal-2021-g/'
+    response = requests.get(html, headers={'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)AppleWebKit'
```

Comment:
do you see how your headers are duplicated? where is the better place to store them?